modules/transactions/src/main/java/org/apache/ignite/internal/tx/TxManager.java

```diff
@@ -149,4 +149,19 @@ CompletableFuture<Void> cleanup(
      * @return Future of all read-only transactions with read timestamp less or equals the given new low watermark.
      */
     CompletableFuture<Void> updateLowWatermark(HybridTimestamp newLowWatermark);
+
+    /**
+     * Registers the infligh update for a transaction.
+     *
+     * @param tx The transaction id.
```

Comment:
txId not tx, but as mentioned below probably not relevant because I believe we should move given method to InternalTransaction.